<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg2</class>
 <widget class="QDialog" name="dlg2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract Features</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="imgsPath_te">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="feature_te">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="imgs_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ImageFolder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="features_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>FeatureFile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="extract_btn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
